### ISSUE TITLE
Weird behaviour of `Open in new tab` in the DevTools.

### ISSUE THREAD

**Issue by emanuele6** [qutebrowser/qutebrowser#5450]
**Version info**:
```
qutebrowser v1.11.1
Git commit: 
Backend: QtWebEngine (Chromium 77.0.3865.129)
Qt: 5.14.2
```
**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
Clicking on `Open in new tab` in the context menu of a `src` link in the DevTools will open the link in the DevTools window. In that window, keybinds won't work and the only ways to go back to the DevTools are either closing the DevTools window and reopening it (closing, not just hiding with `wi`) or using the context menu and clicking on `Back`.  
![qutebrowser_behavioural_bug](https://user-images.githubusercontent.com/20175435/82435146-2718de80-9a94-11ea-8280-e59d8cd47e6b.png)
![qutebrowser_behavioural_bug_part2](https://user-images.githubusercontent.com/20175435/82435181-34ce6400-9a94-11ea-96e9-afaa53745319.png)

I also notice while checking whether this also happened with `--temp-basedir` that this error gets printed:
```
12:10:57 ERROR: Uncaught ReferenceError: DevToolsAPI is not defined
```

**How to reproduce**
* open `qutebrowser`: e.g. `qutebrowser --tempdir`
* go to https://github.com/qutebrowser/qutebrowser
* open the DevTools with `wi`
* right-click on any underlined link
* click on `Open in new tab`
* the error will get printed and the link will open in the DevTools window

-------------------------------------------------
**Note**
I installed `chromium` to check the behaviour there and, when clicking on `Open in new tab` in the DevTools while the DevTools are a separate window, it just opens that link in the browser window, not the DevTools window.

**Comment by The-Compiler** [comment]
This should be fixed in 312e2b49363a4fb38b80e534d81563c0fa87e0e7 which is in the `docked-inspector` branch, as part of #4996. Could you verify that this is the case?

**Comment by emanuele6** [comment]
I'll look into it.
